public/js/lib/colorContraster.js

```diff
@@ -31,12 +31,24 @@ function parseColor(color) {
 
 function parseHex(color) {
   // #ff00de
-  // support for 3 character hex strings would be nice :)
-  // pull request me if you like - noopkat
-  const justColor = color.substring(1, 7);
-  const r = parseInt(justColor.substring(0, 2), 16); // hexToR
-  const g = parseInt(justColor.substring(2, 4), 16); // hexToG
-  const b = parseInt(justColor.substring(4, 6), 16); // hexToB
+  let r; let g; let b; let justColor;
```

Comment:
Since you're making additions, this can be written out more simply:

```suggestion
  let r,  g, b,  justColor;
```